<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>423</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MartianMono Nerd Font</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <family>JetBrainsMono NFM</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="url_check_btn">
       <property name="font">
        <font>
         <family>MartianMono Nerd Font</family>
        </font>
       </property>
       <property name="text">
        <string>Check URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="download_btn">
       <property name="font">
        <font>
         <family>MartianMono Nerd Font</family>
        </font>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MartianMono Nerd Font</family>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrainsMono Nerd Font Mono</family>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QCheckBox" name="yt_search_chkbx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MartianMono Nerd Font</family>
      </font>
     </property>
     <property name="text">
      <string>ytsearch ?</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>160</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Video</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Audio</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
